apps/opencs/model/world/refidadapterimp.cpp

```diff
@@ -431,7 +438,15 @@ void CSMWorld::MiscRefIdAdapter::setData (const RefIdColumn *column, RefIdData&
         InventoryRefIdAdapter<ESM::Miscellaneous>::setData (column, data, index, value);
 }
 
-CSMWorld::NpcColumns::NpcColumns (const ActorColumns& actorColumns) : ActorColumns (actorColumns) {}
+CSMWorld::NpcColumns::NpcColumns (const ActorColumns& actorColumns)
+: ActorColumns (actorColumns),
+  mFlags(std::map<const RefIdColumn *, unsigned int>()),
```

Comment:
This line is kinda pointless, that is what the default map constructor would be doing anyway.